File: add and multiply.cpp
#include <iostream>
using namespace std;
typedef long long ll;
int main()
{
	ll a = 0, b = 0, c = 0, v[5] = {0}, min = 2199999999, max = -299999999;
	short t = 0, i = 0;
	cin>>t;
	
	while(t--)
	{
		cin>>a>>b>>c;
		v[0] = (a + b) * c;
		v[1] = (a + c) * b;
		v[2] = (b + c) * a;
		v[3] = a * c + b;
		v[4] = a * b + c;
		v[5] = b * c + a;
		for(i = 0; i < 6; i++)
		{
			if(v[i] > max)
				max = v[i];
			if(min > v[i])
				min = v[i];
		}
		cout<<min<<" "<<max<<endl;
		min = 2199999999;
		max = -299999999;
	}
	return 0;
}

File: add and multiply.py
n = 0
a = 0
b = 0
c = 0
v = []
n = input()
for i in range(n):
	s = raw_input()
	a = int(s.split(" ")[0])
	b = int(s.split(" ")[1])
	c = int(s.split(" ")[2])
	v.append((a + b) * c)
	v.append((a + c) * b)
	v.append((b + c) * a)
	v.append(a * c + b)
	v.append(a * b + c)
	v.append(b * c + a)
	print str(min(v)) + " " + str(max(v))
	v[:] = []


